app/docker/views/images/import/importimage.html

```diff
@@ -19,7 +19,7 @@
           </div>
           <div class="form-group">
             <div class="col-sm-12">
-              <button type="button" class="btn btn-sm btn-primary" ngf-select ngf-min-size="10" ngf-accept="'application/x-tar,application/x-gzip'" ng-model="formValues.UploadFile"
+              <button type="button" class="btn btn-sm btn-primary" ngf-select ngf-min-size="10" ng-model="formValues.UploadFile"
```

Comment:
```suggestion
              <button type="button" class="btn btn-sm btn-primary" ngf-select ngf-min-size="10" ngf-accept="'application/x-tar,application/x-gzip,application/x-bzip,application/x-xz'" ng-model="formValues.UploadFile"
```
I think it is not a very good idea to make all the files selectable at first place, we still need to add a constraints here. Users can switch to the `All Files` option as they like when they are selecting the file to upload

Reply:
@ArrisLee this change has been tried both by myself and @deviantony before publishing the PR, and it would not work (no effect).

Reply:
Also to add on top of that, the filter capability is actually browser dependent. Whilst this feature is available on Chrome, it is not available at all on Firefox (with or without the `ngf-accept` directive being set).

Hence why the decision was to remove it entirely to offer the same experience across all browsers (although I get your point @ArrisLee and that's why we added the directive originally).

Reply:
@deviantony @jeanfrancoisgratton thanks guys!